src/Reorder/Reorderer.php

```diff
@@ -21,21 +23,40 @@
     /** @var OrderProcessorInterface */
     private $orderProcessor;
 
+    /** @var MoneyFormatterInterface */
+    private $moneyFormatter;
+
+    /** @var Session */
+    private $session;
+
     public function __construct(
         OrderFactoryInterface $orderFactory,
         EntityManagerInterface $entityManager,
-        OrderProcessorInterface $orderProcessor
+        OrderProcessorInterface $orderProcessor,
+        MoneyFormatterInterface $moneyFormatter,
+        Session $session
     ) {
         $this->orderFactory = $orderFactory;
         $this->entityManager = $entityManager;
         $this->orderProcessor = $orderProcessor;
+        $this->moneyFormatter = $moneyFormatter;
+        $this->session = $session;
     }
 
     public function reorder(OrderInterface $order, ChannelInterface $channel): OrderInterface
     {
         $reorder = $this->orderFactory->createFromExistingOrder($order, $channel);
         assert($reorder instanceof OrderInterface);
 
+        if ($reorder->getTotal() !== $order->getTotal()) {
+            $formattedTotal = $this->moneyFormatter->format($order->getTotal(), $order->getCurrencyCode());
+
+            $this->session->getFlashBag()->add('info', [
+                'message' => 'sylius.reorder.items_price_changed',
+                'parameters' => ['%order_total%' => $formattedTotal]
+            ]);
+        }
```

Comment:
Maybe it could be a separated service? `Reorderer` should only take care about reordering, possible notification should be processed by something like `ReorderingChangesNotifier`

Reply:
It's alredy separated in further pull requests.